easybuild/easyconfigs/c/cairo/cairo-1.14.10-GCCcore-6.3.0.eb

```diff
@@ -0,0 +1,43 @@
+easyblock = 'ConfigureMake'
+
+name = 'cairo'
+version = '1.14.10'
```

Comment:
@boegel Don't want to open yet another can of worms, but wondering about the `GLib` `versionsuffix` that some of the predecessors have.